apps/reference-backend/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.3'
 
 services:
   db_reference_backend:
-    image: public.ecr.aws/bitnami/postgresql:11.12.0
+    image: postgres:11.12-alpine
```

Comment:
this might cause us problems with pull limits from dockerhub

Reply:
Reverted back to bitnami